assets/state-vgpu-device-manager/0600_daemonset.yaml

```diff
@@ -62,7 +76,17 @@ spec:
         hostPath:
           path: /sys
           type: Directory
+      - name: driver-install-dir
```

Comment:
Note: the `reconfigure_mig.sh` script assumes that the host root fs is mounted at `/host` in the container (it chroot's into the host rootfs to stop / start systemd services when drivers are installed directly on the host). So we need to add a volume for the host root.

Reply:
Great catch and thank you for the explanation! I added the `/host` mount.